README.md

```diff
@@ -39,37 +39,6 @@ Compliance-to-Policy (C2P) bridges Compliance as Code and Policy as Code. C2P ta
  <img src="http://img.youtube.com/vi/wC9QEUoCcuo/mqdefault.jpg" alt="Watch the video"/>
 </a>
 
-## Community meetings and communications
```

Comment:
I think leaving the information in the README.md is important for discoverability even though the information will be duplicated somewhat.